Unreal Launcher/Unreal Launcher/Project.cs

```diff
@@ -33,7 +33,7 @@ public class Project
 		public ProjectLaunchSettings LaunchSettings = new ProjectLaunchSettings();
```

Comment:
Assuming this doesn't need to be publicly settable, this would be better as:
`private ProjectLaunchSettings _launchSettings = new ProjectLaunchSettings();`
`public ProjectLaunchSettings LaunchSettings => _launchSettings;`